services/property-svc/stories/PropertyValueCRUD_test.go

```diff
@@ -635,7 +635,7 @@ func TestCreateAttachAddOptionAttachSelectProperty(t *testing.T) {
 				RemoveOptions: nil,
 				UpsertOptions: []*pb.UpdatePropertyRequest_SelectData_SelectOption{
 					{
-						Id:          "",
+						Id:          uuid.NewString(),
```

Comment:
Nowhere else we require the frontend to provide a would-be uuid for a new resource. I don't think this should be the exception.

Reply:
Thanks, didn't think about it